frontend/src/assets/css/variables.css

```diff
@@ -1,5 +1,9 @@
 :root {
-  --bt-blue-gray: #90a3bf;
-  --bt-input: #d5dce8;
-  --hover: #274F8D;
+    --bt-blue-gray: #90a3bf;
```

Comment:
my fail, this file should be .scss and imported to styles.scss

Reply:
So i just need to change extension to scss and leave my variables as it is?